keps/sig-node/3386-kubelet-evented-pleg/README.md

```diff
@@ -341,6 +344,10 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - Feature implemented behind a feature flag
 - Existing `node e2e` tests around pod lifecycle must pass
 
+#### Beta
+- Add E2E Node Conformance presubmit job in CI 
+- Add E2E Node Conformance periodic job in CI
```

Comment:
this would require containerd support I believe. Evented PLEG related support is in containerd 1.7, but I think we have a testgrid testing against main branch of containerd, which should be sufficient.